charts/nri-bundle/README.md

```diff
@@ -132,6 +132,8 @@ newrelic-infrastructure:
       selector: "newrelic.com/custom-ksm=true"
 ```
 
+For more information on supported KSM version visit the [requirements documentation](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements#reqs)
```

Comment:
this is not in the template, therefore it will be lost

Reply:
It is [already included](https://github.com/newrelic/helm-charts/pull/994/files#diff-8f841954c94909bbe80e2b5234f062bc78bd35d2a60430db63a03c79bb3a17abR137)